test/documentation/DoxygenLayout.xml

```diff
@@ -0,0 +1,228 @@
+<doxygenlayout version="1.0">
```

Comment:
Can we use this file as-is from the SeqAn3 submodule instead of copying it?

Reply:
No because in seqan the structure includes a Howto, about, code of conduct, tutorial etc which would have dead links if we include it

Reply:
Ok, we can think about replacing it later, if we add those things.